apps/web/src/auth.ts

```diff
@@ -96,19 +96,26 @@ const providers: Provider[] = [
         },
       });
 
+      // Send appropriate email based on whether user exists
       if (user) {
-        await getEmailClient(user.locale ?? undefined).sendTemplate(
-          "LoginEmail",
-          {
-            to: email,
-            props: {
-              magicLink: absoluteUrl("/auth/login", {
-                magicLink: url,
-              }),
-              code: token,
-            },
+        // Send login email
+        await getEmailClient(user.locale ?? undefined).sendTemplate("LoginEmail", {
+          to: email,
+          props: {
+            magicLink: absoluteUrl("/auth/login", {
+              magicLink: url,
+            }),
+            code: token,
           },
-        );
+        });
+      } else {
+        // Send registration email
```

Comment:
If the user doesn't exist we shouldn't send **any** email. It is neither desirable nor possible to complete a registration from the login form.